client/src/app/project/project.component.html

```diff
@@ -6,12 +6,12 @@
   <app-uuid [uuid]="projectUuid"></app-uuid>
   <h1>{{getProjectName()}}</h1>
   <br/>
-  <mat-tab-group [selectedIndex]="lookupTabIndex(selectedProjectTabKey)">
+  <mat-tab-group [selectedIndex]="lookupTabIndex(selectedMainTabKey)" (selectedTabChange)="mainTabChange($event)">
     <mat-tab [label]="tabConfig[0].label">
       <div class="vf-u-margin__top--xxl">
         <app-project-summary (tabChange)="projectTabChange($event)" [project]="project"></app-project-summary>
         <a class="vf-button vf-button--sm {{upload ? '' : 'vf-button--primary'}}" role="button"
-           routerLink="/projects/{{project?.uuid?.uuid}}/{{selectedProjectTabKey}}">Edit Project</a>
+           routerLink="/projects/{{project?.uuid?.uuid}}/edit" [queryParams]="{tab: selectedProjectTabKey}">Edit Project</a>
```

Comment:
queryParams attribute in a separate line.

Reply:
Done but please, let's just use a code formatter. These kind of comments don't need to be part of a PR